main.ledger

```diff
@@ -1802,6 +1802,12 @@
     Liabilities:Reimbursement:Zach Latta
     ; Receipt: e1e9fb6f5efb4480793bdc16b2470ccd.pdf
 
+2016/03/20 IRCCloud
+    Expenses:Services:Software                           $5.00
+    Assets:Bank:Checking
+    ; Receipt: bec76e76e07037674b22877534238ba3.pdf
+    ; TODO why is this so old
```

Comment:
I think I found and old receipt. This is reimbursements though, so it shouldn't show in the bank at all.